From a8eb11f56089844fc3a09548044afd163541d102 Mon Sep 17 00:00:00 2001
From: Simon Pena <spena@igalia.com>
Date: Fri, 9 May 2025 15:14:36 +0100
Subject: [PATCH 6/7] [WPE] Android: build WebDriver as a shared library
 https://bugs.webkit.org/show_bug.cgi?id=292770

Reviewed by NOBODY (OOPS!).

When building WebDriver on Android, make sure it uses a shared library.

* Source/WebDriver/CMakeLists.txt:
---
 Source/WebDriver/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WebDriver/CMakeLists.txt b/Source/WebDriver/CMakeLists.txt
index d2339c975b5b..ae24a7650696 100644
--- a/Source/WebDriver/CMakeLists.txt
+++ b/Source/WebDriver/CMakeLists.txt
@@ -62,6 +62,7 @@ list(APPEND WebDriver_SOURCES ${WebDriver_DERIVED_SOURCES_DIR}/WebDriverAtoms.cp
 
 if (ANDROID AND (PORT STREQUAL WPE))
     set(WebDriver_TARGET_TYPE LIBRARY)
+    set(WebDriver_LIBRARY_TYPE SHARED)
 else ()
     set(WebDriver_TARGET_TYPE EXECUTABLE)
 endif ()
-- 
2.43.0

